### PR TITLE
Limited .go and .js files to 80 columns 

### PR DESCRIPTION
**Pull request by atmiguel**

### CODE REVIEW

File: databaseutil/databaseutil.go [hunk 1]
```diff
@@ -49,16 +63,19 @@ func CreateUser(displayName string, emailAddress string, password string) (int64
 
 func AuthenticateUser(emailAddress string, password string) (bool, error) {
 	sqlQuery := `
-		SELECT password FROM users WHERE email_address = $1
-	`
+		SELECT password FROM users
+		WHERE email_address = $1`
```
Comment (zingales) [inline databaseutil/databaseutil.go]: Is this SQL formatting not 80 char rule ?

File: static/js/login_or_signup.js [hunk 2]
```diff
@@ -3,22 +3,23 @@ var getInputField = function($form, field) {
 };
 
 var getInputFields = function($form, fields) {
-    return fields.map(
-        (field) => getInputField($form, field));
+    return fields.map((field) => getInputField($form, field));
 };
 
 var checkFormValidity = function($form, fields) {
-    return getInputFields($form, fields).every(
```
Comment (zingales) [inline static/js/login_or_signup.js:11]: Curious why remove this cascade methodology?

File: static/js/login_or_signup.js [hunk 3]
```diff
@@ -3,22 +3,23 @@ var getInputField = function($form, field) {
...
-        ($field) => $field.get(0).checkValidity());
+    var $inputFields = getInputFields($form, fields);
+
```
Comment (zingales) [inline static/js/login_or_signup.js]: Consider removing this newline.

File: static/js/login_or_signup.js [hunk 4]
```diff
@@ -3,22 +3,23 @@ var getInputField = function($form, field) {
...
-        (formData, $field) => {
-            formData[$field.attr('name')] = $field.val();
-            return formData;
-        },
-        {});
+    var $inputFields = getInputFields($form, fields);
+
+    return $inputFields.reduce((formData, $field) => {
+        formData[$field.attr('name')] = $field.val();
+
```
Comment (zingales) [inline static/js/login_or_signup.js]: consider removing this newline.

File: static/js/login_or_signup.js [hunk 5]
```diff
@@ -51,103 +50,87 @@ $(function() {
...
-                            var value = $field.val();
-                            var trimmedValue = $.trim(value);
-
-                            $field.val(trimmedValue);
-                        });
+        var $inputFields = getInputFields(
+            formMetadata.$form,
+            formMetadata.fields);
+
+        $inputFields.forEach(($field) => {
```
Comment (zingales) [inline static/js/login_or_signup.js:66]: I really like that you did $inputFields here. Makes the code more readable.

File: static/js/login_or_signup.js [hunk 6]
```diff
@@ -51,103 +50,87 @@ $(function() {
...
-                (response) => {
-                    alert(response);
-                }, 
-                'text');
+                touchAllFields(formMetadata.$form, formMetadata.fields);
+            }
         });
+    }
+
+    installSubmitClickHandler(signupFormMetadata, (formDataAsJsonString) => {
```
Comment (zingales) [inline static/js/login_or_signup.js]: Do you want to rename these as attach? Or do you want me to do that in a different pull request?

File: static/js/login_or_signup.js (hunk 2 above)
Comment (atmiguel) [reply static/js/login_or_signup.js:11]: It seemed the cleanest, most readable solution to not have this line be over 80 chars long

File: static/js/login_or_signup.js (hunk 6 above)
Comment (atmiguel) [reply static/js/login_or_signup.js]: I'll make that change

**Review by zingales — APPROVED** [review]
